### ISSUE TITLE
Autocalc not working as expected after inserting rows

### ISSUE THREAD

**Issue by oaklandgit** [andmarti1424/sc-im#695]
Wonderful piece of software! But I am encountering the following issue:

**Steps to reproduce:**
1. Confirm that `autocalc=1` in settings
2. Enter values into 3 cells (e.g. A0`10`, A1`20`, A2`30`)
3. On 4th cell enter formula `=@SUM(A0:A2)`. Cell will show `60` as expected
4. With `or` (open row) command, add a cell between A1 and A2 and enter a value (so now you have e.g. `10`, `20`, `25`, `30`)
5. The formula row now properly updated to `=@SUM(A0:A3)` but the value still shows `60` instead of `85` as expected

In short: the formula seems to be aware of the added row, but the calculation doesn't update accordingly.

**Comment by spearman** [comment]
I came to make the same issue. I haven't used sc-im much so maybe I am missing something, but when I use `fcopy` to copy a formula, only the value of the original cell will update when changing inputs. If I close the file and reload, then it works as expected (changing inputs updates the values in the copied formula cells).

**Comment by oaklandgit** [comment]
+1 @spearman. I need to close and reload for the calculations to take effect.

**Comment by andmarti1424** [comment]
@oaklandgit The result in my case gets updated correctly to 85. What version of sc-im are you using?

**Comment by andmarti1424** [comment]
Seems outdated. Could you please retry with the version on dev branch? I could not managed to reproduce it with it. Thanks.

**Comment by oaklandgit** [comment]
OK thanks -- I'm not well-versed on how to configure a Makefile for my system so I may wait until the newer version is available through Homebrew.

**Comment by andmarti1424** [comment]
@oaklandgit , could you test this with a newer version? If that's the case, please report. thanks!

**Comment by oaklandgit** [comment]
@andmarti1424 Could you provide steps for building from dev branch for M1 macs? Or update Homebrew with the latest? (Sorry -- I am still learning how to build from source.)

**Comment by oaklandgit** [comment]
Thank you @andmarti1424 . This version (main branch) seems to work!
